docs/components/modeler/bpmn/bpmn-coverage.md

```diff
@@ -140,15 +140,19 @@ import CompensationSvg from './assets/bpmn-symbols/compensation.svg'
 
 ## Data
 
+:::note
+`DataObject` and `DataStore` are supported by Camunda for modeling purposes only. Camunda does not support any additional data features the BPMN standard defines.
```

Comment:
We could also frame this as "we don't support BPMN standard IO mappings (but of course allow you to map IO (in our very simple, CAmunda style) [link to our documentation for that]". This may read as "We do not support data features".

Reply:
How about "DataObject` and `DataStore`, like other BPMN standard IO mappings, are supported by Camunda for modeling purposes only."? We could leave out "Camunda does not support any additional data features the BPMN standard defines."

Reply:
Whoops I think we forgot about this @nikku -- WDYT here? CC @aleksander-dytko

Reply:
> We could leave out "Camunda does not support any additional data features the BPMN standard defines."

Yea, we could leave it out, or link to our own support for I/O. Probably easiest to leave it out.